app/model/repository/Exercises.php

```diff
@@ -47,22 +59,139 @@ public function searchByName(?string $search): array {
     }
 
     return $this->searchHelper($search, function ($search) {
-      $idsQueryBuilder = $this->em->createQueryBuilder()->addSelect("l.id")->from(LocalizedExercise::class, "l");
```

Comment:
The `searchByName` function is used somewhere else than in the `Exercises:default` action? Or why it stayed here?

Reply:
It is definitely used in tests since it is a convenient way to get a user. I have decided to keep it rather than fixing all the test (plus using pagination just to fetch one user by the name is a bit tedious).

Reply:
Ok...